_rules/inline-link-in-paragraph-has-distinguishable-border-36f116.md

```diff
@@ -0,0 +1,159 @@
+---
+id: 36f116
+name: Inline link has distinguishable border
+rule_type: atomic
+description: |
+  This rule checks that inline links have a border that distinguishes them from the surrounding text
+accessibility_requirements:
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Brian Bors
+    - Carlos Duarte
+  previous authors:
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to any [visible][] HTML element that is a [semantic link][], where the closest element (including itself) that generates an [inline box][] is part of a [line box][] that has [inline boxes][inline box] generated from an element containing [visible][] [text nodes][text node] that do not have a [semantic link][] as a [descendant][] in the [flat tree][].
+
+**Note:** The test targets are visible link elements that are part of a block of content that is displayed inline.
+
+## Expectation
+
+Each target element has a [visible][] [border](https://drafts.csswg.org/css-backgrounds/#borders).
```

Comment:
You're putting quite a bit of weight on "visible" with this. I really like this, but I think it might be useful to address https://github.com/act-rules/act-rules.github.io/issues/1253. I won't vote it down if we don't, but I would much prefer it if we did. For this case for example, we could say that CSS properties with a color are tested by changing the alpha to 0 for all colors.

Reply:
Where would we add that? A note in this rule (and any other rule that uses visible) or stack all the ways that visibility can be tested in the visible definition?